sys/kern/sleepq.c

```diff
@@ -324,7 +324,8 @@ int sleepq_wait_timed(void *wchan, const void *waitpt, mtx_t *mtx,
     callout_schedule(&td->td_slpcallout, timeout);
   }
 
-  td->td_flags |= (timeout > 0) ? TDF_SLPTIMED : TDF_SLPINTR;
+  td->td_flags |= TDF_SLPINTR;
+  td->td_flags |= (timeout > 0) ? TDF_SLPTIMED : 0;
```

Comment:
`if (timeout > 0) td->td_flags |= TDF_SLPTIMED` is more readable and or with 0 is nop.

Reply:
Right, dunno why I made it this way.